contact/forms.py

```diff
@@ -27,9 +27,9 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         # Update the required fields
-        if self.required_fields:
+        if self.Meta.required_fields:
```

Comment:
I feel like the logic is back to front here. If `required_fields` defaults to an empty list instead of None you can simplify this to:
```
for field in self.Meta.required_fields:
    self.fields[field].required = True
```

Reply:
Might also be better defining `required_fields` on the class itself, then you dont need to access it via `self.Meta`